qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -195,7 +219,8 @@ def clifford_2_qubit_circuit(self, num):
             qc.z(1)
         return qc
 
-    def _unpack_num(self, num, sig):
```

Comment:
In lines 206-207 above - can replace:
```
                qc._append(VGate(), [qr[1]], [])
                qc._append(VGate(), [qr[1]], [])
```
by:
`                qc._append(WGate(), [qr[1]], [])
`

Reply:
One can replace 2 VGate by one WGate. VGate is of order 3, and WGate is the inverse of VGate.
Lines 163-164 should be the same as line 159 (case k0==2 is the same as case k1==2)

![image](https://user-images.githubusercontent.com/46566946/180923184-ed9e6066-d408-4a5b-847a-0a32f0cbbf57.png)